test/unit/dinero.spec.js

```diff
@@ -684,4 +684,44 @@ describe('Dinero', () => {
       )
     })
   })
+  describe('#minimum', () => {
+    test('should return the minimum dinero from an array of dinero objects', () => {
+      const minimumDineroExpected = Dinero({ amount: 50, precision: 2 })
+      const actualMinimum = Dinero.minimum([
+        minimumDineroExpected,
+        Dinero({ amount: 100, precision: 2 }),
+        Dinero({ amount: 150, precision: 2 })
+      ])
+      expect(actualMinimum.toJSON()).toEqual(minimumDineroExpected.toJSON())
+    })
+    test('should return the minimum dinero from an array of dinero objects, with different precision on those objects', () => {
+      const minimumDineroExpected = Dinero({ amount: 50, precision: 3 })
```

Comment:
I would go with amount 500 (which is numerically greater than 100 and 150) because this further proves that `Dinero.minimum` takes precision into account.

```suggestion
      const minimumDineroExpected = Dinero({ amount: 500, precision: 3 })
```